blockchain/vm/eips.go

```diff
@@ -44,14 +46,14 @@ func EnableEIP(eipNum int, jt *JumpTable) error {
 // - Define SELFBALANCE, with cost GasFastStep (5)
 func enable1884(jt *JumpTable) {
 	// Gas cost changes
+	jt[SLOAD].constantGas = params.SloadGasEIP1884
 	jt[BALANCE].constantGas = params.BalanceGasEIP1884
 	jt[EXTCODEHASH].constantGas = params.ExtcodeHashGasEIP1884
-	jt[SLOAD].constantGas = params.SloadGasEIP1884
 
 	// Computation cost changes
+	jt[SLOAD].computationCost = params.SloadComputationCostEIP1884
 	jt[BALANCE].computationCost = params.BalanceComputationCostEIP1884
 	jt[EXTCODEHASH].computationCost = params.ExtCodeHashComputationCostEIP1884
```

Comment:
Could you explain this part of the code? These lines will overwrite the previous values.  Also, I cannot find them in go-eth.

Reply:
@winnie-byun Uhm.. maybe it's because it's about computation cost. It's similar to gas cost part. 
line 49
```
jt[SLOAD].constantGas = params.SloadGasEIP1884
jt[BALANCE].constantGas = params.BalanceGasEIP1884
jt[EXTCODEHASH].constantGas = params.ExtcodeHashGasEIP1884
```

Reply:
I misunderstood the code. One was setting `constantGas` and the other one is setting `computationCost`.